doc/doxygen/custom.js

```diff
@@ -3,6 +3,7 @@ $( document ).ready( function() {
     $('.doxygen-generated-exception-message').each(function(i, obj) {
 	var s = $(this).html();
 	s=s.replace(/\\n|std::endl/g,"<br/>");
+	s=s.replace(/\\"/g,"'");
```

Comment:
Is it correct that that matches *internal* double quotes spelled as `\"` inside the string, and just replaces them by single quotes?